knowledge/pkg/datastore/documentloader/ocr/openai/openai.go

```diff
@@ -290,10 +293,34 @@ func (o *OpenAIOCR) SendImageToOpenAI(ctx context.Context, base64Image string) (
 		return "", fmt.Errorf("OpenAI OCR error sending request(s): %w", err)
 	}
 
+	err = os.WriteFile("/tmp/openai_response.json", body, 0777)
+	if err != nil {
+		return "", fmt.Errorf("error writing openai response to file: %w", err)
+	}
```

Comment:
Was this for debugging? If so, it should be removed.

Reply:
:sweat_smile: no idea how that could slip through